content/en/guides/models/track/runs/color-code-runs.md

```diff
@@ -73,7 +73,7 @@ Consider the following configuration:
 
 {{< img src="/images/track/color-coding-runs.png" alt="" >}}
 
-**Metric** is set to `"Accuracy"` (abbreviated as `"acc"`). **Y value** is set to `"Max"`, meaning the maximum value of the accuracy metric is used to determine the color coding for the runs. 
+**Metric** is set to `"Accuracy"` (abbreviated as `"acc"`). **Y value** is set to `"Max"`, meaning the maximum value of the accuracy metric is used to determine the color of a run.
```

Comment:
```suggestion
- **Metric** is set to `"acc"`, which represents accuracy.
- **Y value** is set to `"Max"`, which determines the color of a run based on the maximum accuracy logged during the run.
```
It is confusing to mark up `"Accuracy"` when that's not what the metric is called.